src/qt/addressbookpage.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2011-2015 The Bitcoin Core developers
-// Copyright (c) 2015-2017 The Bitcoin Unlimited developers
+// Copyright (c) 2015-2018 The Bitcoin Unlimited developers
```

Comment:
looks like a lot updated to 2018?

Reply:
the script update the copyright year according to the last time it was modified. 

In fact if you look at the git history of the above file, the last commit was: 

```
commit 5502d2fdcabeaf0ddf83aaca4dd6be2a902fb727
Author: Andrew Stone <g.andrew.stone@gmail.com>
Date:   Mon Jul 30 07:55:29 2018 -0400

    fix shadowing and other warnings (#1227)
    
    * fix shadowing and other warnings
```

which as you can see has been committed on 2018